src/common/set.c

```diff
@@ -3138,9 +3150,16 @@ util_pool_create_uuids(struct pool_set **setp, const char *path,
 	/* attributes cannot be NULL for local replicas */
 	ASSERT(remote || attr != NULL);
 
+	struct pool_attr adj_attr;
 	int flags = MAP_SHARED;
 	int oerrno;
 
+	if (attr != NULL) {
+		adj_attr = *attr;
+		util_adjust_pool_attr(&adj_attr);
```

Comment:
This is odd place to put something like that. util_header_create seems better (thx Janek for suggestion).